eng/Directory.Build.Data.targets

```diff
@@ -69,6 +69,16 @@
     <None Condition="Exists('$(MSBuildProjectDirectory)/../README.md')" Include="$(MSBuildProjectDirectory)/../README.md" Pack="true" PackagePath=""/>
   </ItemGroup>
 
+  <!-- Enables CodeCoverage -->
+  <ItemGroup Condition="'$(IsTestProject)' == 'true'">
+    <PackageReference Include="coverlet.msbuild">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <Import Condition="'$(CodeCoverageArgs)' != ''"  Project="$(MSBuildThisFileDirectory)\DeterministicBuild.targets" />
```

Comment:
I don't see CodeCoverageArgs being set as an msbuid property should this be CollectCoverage?

Reply:
Am setting it at the point where we run dotnet test, but I see what you mean. I'll do that.